Sources/Core/Shared/Operation.swift

```diff
@@ -379,11 +379,6 @@ public extension Operation {
             }
         })
 
-        // The evaluator waits for all condition dependencies
-        conditions
```

Comment:
Evaluation or any of its dependencies should not run until after previous mutually exclusive operation has finished.